api/v1/sp.go

```diff
@@ -43,8 +50,19 @@ type StorageProviderIDs struct {
 //	@Produce		json
 //	@Success		200	{object}	ResponseEnvelope{response=GetStorageProvidersResponse}
 //	@Router			/sp/approve [post]
-func handleApproveStorageProviders(c echo.Context) error {
-	return c.JSON(http.StatusNotImplemented, map[string]string{})
+func (a *apiV1) handleApproveStorageProviders(c echo.Context) error {
+	var storageProviderIds Sps
```

Comment:
For any of those methods where an array of `spids` is passed in, i think it's better if we just make the body an array of spids (int) instead of decoding to a JSON object 

(i.e, right now it must look like
```json
[
   { "sp_id": 123 },
   { "sp_id": 456 }
]
```

But much simpler if the body of the request can simply be `[123, 456]`